dependencies.gradle

```diff
@@ -0,0 +1,47 @@
+ *       Not available at all for mods depending on this mod, only visible at runtime for this mod
+ *  - devOnlyNonPublishable("g:n:v:c"): a combination of runtimeOnlyNonPublishable and compileOnly for dependencies present at both compiletime and runtime,
+ *       but not published as Maven dependencies - useful for RFG-deobfuscated dependencies or local testing
+ *  - runtimeOnly("g:n:v:c"): if you don't need this at compile time, but want it to be present at runtime
+ *       Available at runtime for mods depending on this mod
+ *  - annotationProcessor("g:n:v:c"): mostly for java compiler plugins, if you know you need this, use it, otherwise don't worry
+ *  - testCONFIG("g:n:v:c") - replace CONFIG by one of the above (except api), same as above but for the test sources instead of main
+ *
+ *  - shadowImplementation("g:n:v:c"): effectively the same as API, but the dependency is included in your jar under a renamed package name
+ *       Requires you to enable usesShadowedDependencies in gradle.properties
+ *
+ *  - compile("g:n:v:c"): deprecated, replace with "api" (works like the old "compile") or "implementation" (can be more efficient)
+ *
+ * You can exclude transitive dependencies (dependencies of the chosen dependency) by appending { transitive = false } if needed,
+ * but use this sparingly as it can break using your mod as another mod's dependency if you're not careful.
+ *
+ * To depend on obfuscated jars you can use `devOnlyNonPublishable(rfg.deobf("dep:spec:1.2.3"))` to fetch an obfuscated jar from maven,
+ * or `devOnlyNonPublishable(rfg.deobf(project.files("libs/my-mod-jar.jar")))` to use a file.
+ *
+ * Gradle names for some of the configuration can be misleading, compileOnlyApi and runtimeOnly both get published as dependencies in Maven, but compileOnly does not.
+ * The buildscript adds runtimeOnlyNonPublishable to also have a runtime dependency that's not published.
+ *
+ * For more details, see https://docs.gradle.org/8.0.1/userguide/java_library_plugin.html#sec:java_library_configurations_graph
+ */
+dependencies {
+    compileOnly("curse.maven:ars-magica-2-67313:2280862") {transitive = false}
+    compileOnly("curse.maven:biomes-o-plenty-220318:2499612") {transitive = false}
+    compileOnly("curse.maven:chromaticraft-235590:4721192") {transitive = false}
+    compileOnly("curse.maven:extrabiomesxl-60041:2273301") {transitive = false}
+    compileOnly("curse.maven:growthcraft-community-edition-235092:4464624") {transitive = false}
+    compileOnly("curse.maven:highlands-221226:2227924") {transitive = false}
+    compileOnly("curse.maven:mystcraft-224599:2417042") {transitive = false}
+    compileOnly("curse.maven:thaumcraft-223628:2227552") {transitive = false}
+    compileOnly("curse.maven:undergroundbiomesconstructs-72744:2304497") {transitive = false}
+    compileOnly("curse.maven:vampirism-become-a-vampire-233029:2313875") {transitive = false}
```

Comment:
There is no need to specify `transitive = false` on any CurseMaven dep, it only provides the specified dependency.